content/en/blog/_posts/2024-03-18-windows-ops-readiness.md

```diff
@@ -0,0 +1,78 @@
+slug: intro-windows-ops-readiness
+---
+
+**Authors:** Jay Vyas (Tesla), Amim Knabben (Broadcom), and Tatenda Zifudzi (AWS)
+
+# Introducing the Windows Operational Readiness Specification
+
+Since Windows support [graduated to stable](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.14/release-notes-draft.md) with 1.14 in 2019, the capability to run Windows workloads has consistently been a major differentiator for Kubernetes distributions in the enterprise. However, with more Windows workloads being migrated to Kubernetes and new Windows features being continuously released, it became challenging to test Windows worker nodes in an effective and standardized way.
+
+Some notable examples brought to the attention of SIG Windows were:
+
+- An issue with [loadBalancerSourceRanges](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.29/#:~:text=loadBalancerSourceRanges) functionality not operating correctly on Windows nodes, detailed in a GitHub issue [kubernetes/kubernetes#120033](https://github.com/kubernetes/kubernetes/issues/120033).
+- Reports of functionality issues with Windows features, such as “[GMSA](https://learn.microsoft.com/en-us/windows-server/security/group-managed-service-accounts/group-managed-service-accounts-overview) isn’t working, why?”, discussed in [microsoft/Windows-Containers#44](https://github.com/microsoft/Windows-Containers/issues/44).
+- Challenges developing networking policy tests that could objectively evaluate Container Network Interfaces (CNIs) across different operating system configurations, as discussed in [kubernetes/kubernetes#97751](https://github.com/kubernetes/kubernetes/issues/97751)
+
+The SIG Windows community therefore recognized the need for a tailored solution to ensure Windows nodes' operational readiness *before* their deployment into production environments. Thus, the [Windows Operational Readiness Specification](https://github.com/kubernetes/enhancements/blob/d60c6149551b7f852c16842fb952a499c2cc4315/keps/sig-windows/2578-windows-conformance/README.md) was born.
+
+# Can’t we just run the Conformance tests?
+
+The Kubernetes project contains a set of [conformance tests](https://github.com/kubernetes/kubernetes/blob/master/test/conformance/testdata/conformance.yaml), which are standardized tests designed to ensure that a Kubernetes cluster meets the required Kubernetes specifications.
+
+However, these tests were originally defined at a time when Linux was the *only* operating system compatible with Kubernetes, and thus, they were not easily extendible for use with Windows. Given that Windows workloads, despite their importance, account for a smaller portion of the Kubernetes community, it was important to ensure that the primary conformance suite relied upon by many Kubernetes distributions to certify Linux conformance, didn't become encumbered with Windows specific features or enhancements such as GMSA or multi-operating system kube-proxy behavior.
```

Comment:
We might want to mention that, as a project, we value being able to certify conformance without forcing a closed-source licence on a certified distribution or service that doesn't plan to actually offer Windows.